doc/source/quickstart.rst

```diff
@@ -83,6 +93,13 @@ same packages but without ``lib32-`` or ``-multilib``::
 Installing Android SDK
 ~~~~~~~~~~~~~~~~~~~~~~
 
+.. warning::
+   python-for-android is often picky about the **SDK/NDK versions.**
```

Comment:
We have `pythonforandroid.recommendations` but don't do much with it. Maybe we should have a cli option to tell the user what versions to install for the given p4a version. Just noting that for information though, no change needed.